schemas/mw-article-rename.js

```diff
@@ -0,0 +1,48 @@
+{
+    "title": "MediaWiki Article Rename",
+    "description": "Represents a MW Article rename event",
+    "type": "object",
+    "properties": {
+        // global event fields
+        "uri": {
+            "type": "string",
+            "format": "uri",
+            "description": "the unique URI identifying the event"
+        },
+        "request_id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$",
+            "description": "the unique UUID v1 ID of the event derived from the X-Request-Id header"
+        },
+        "dt": {
+            "type": "string",
+            "format": "date-time",
+            "description": "the time stamp of the event, in ISO8601 format"
+        },
+        "domain": {
+            "type": "string",
+            "description": "the domain the event pertains to"
+        },
+        // event-specific fields
+        "title": {
+            "type": "string",
+            "description": "the new title of the article"
+        },
+        "old_title": {
+            "type": "string",
+            "description": "the old title of the article"
+        },
+        "page_id": {
+            "type": "integer",
+            "minimum": 1,
+            "description": "the page ID of the renamed article"
+        },
+        "summary": {
+            "type": "string",
+            "description": "the summary comment left by the user"
+        }
+    },
+    "required": [
+        "uri", "request_id", "dt", "domain", "title", "old_title", "page_id"
+    ]
+}
```

Comment:
`old_revision` and `new_revision` fields might be helpful too, because when article is moved, a new revision of an old title with a redirect is created, so it wouldn't be obvious, which revision is a base for the move.